requirements.txt

```diff
@@ -1,14 +1,14 @@
 # Functional Dependencies
-panther-analysis-tool==0.2.1
+panther-analysis-tool==0.3.0
 policyuniverse==1.3.2.2
 
 # CI Dependencies
 bandit==1.6.2
 mypy==0.770
-pylint==2.4.4
-yapf==0.29.0
+pylint==2.5.2
+yapf==0.30.0
 ## The following requirements were added by pip freeze:
-astroid==2.3.3
+astroid==2.4.1
 boto3==1.12.24
```

Comment:
Pin `boto3` to 1.12.22 to match the version in Lambda (and the version pinned in Panther itself)